### ISSUE TITLE
Is this project still maintained?

### ISSUE THREAD

**Issue by cipher-fox** [ntegral/nestjs-sentry#111]
There are certain pull requests pending and some of them do not carry major improvements that could break the code.

If any help is needed to maintain the project, feel free to contact me. 👨‍💻⚒️

**Comment by finnatsea** [comment]
Where are we on this?

**Comment by namoscato** [comment]
It looks like [travelerdev](https://github.com/travelerdev/nestjs-sentry) is a pretty active fork. @zackdotcomputer, can you confirm?

**Comment by zackdotcomputer** [comment]
@namoscato yup - I'm using that fork in production both for day job work and a couple side projects. I'm maintaining it actively and open to PRs.

**Comment by AdamGerthel** [comment]
@zackdotcomputer wouldn't it be better if @hardydex gave you ([travelerdev](https://github.com/travelerdev)?) ownership or co-ownership of this repository? I wouldn't mind helping out on a specific issue, but I'm also very hesitant to rely on a small fork of a stagnant project. If you took over maintenance then maybe the project could live longer.

**Comment by rohanrajpal** [comment]
Thanks a lot @zackdotcomputer ! Switched to your fork

**Comment by jschroed91** [comment]
Thank you @zackdotcomputer and thanks @namoscato for tracking that down for us :)

**Comment by ntegral** [comment]
This library is still being maintained. Thanks.  An update was pushed today.
